hamilton/function_modifiers/metadata.py

```diff
@@ -166,3 +166,91 @@ def decorate_node(self, node_: node.Node) -> node.Node:
         new_tags = node_.tags.copy()
         new_tags.update(self.tag_mapping.get(node_.name, {}))
         return tag(**new_tags).decorate_node(node_)
+
+
+# These represent a generic schema type -- E.G. one that will
+# be supported across the entire set of usable dataframe/dataset types
+# Eventually we'll be integrating mappings of these into the registry,
+# but for now this serves largely as a placeholder/documentation
+GENERIC_SCHEMA_TYPES = (
+    "int",
+    "float",
+    "str",
+    "bool",
+    "dict",
+    "list",
+    "object",
+    "datetime",
+    "date",
+)
```

Comment:
where is this used?

Reply:
Its not -- I'll comment it out but leave it in for now. See comment above.